src/Handler/TagHandlerInterface.php

```diff
@@ -6,5 +6,20 @@
 
 interface TagHandlerInterface
 {
+    /**
+     * @deprecated Use addTags()
+     */
     public function addTagHeaders(Response $response, array $tags);
+
+    /**
+     * @param array $tags
+     *
+     * @return $this
+     */
+    public function addTags(array $tags);
```

Comment:
Adding methods to interfaces is a BC break (existing implementations of this interface will break on upgrade).

We could cheat, ignore `$response` in the implementation, and just call `parent::addTags()`. 
Ideally, we would add another interface, that has the correct method, and have it implemented the current one. But what would we call it ? The good name is already taken :)

Reply:
ResponseTaggerInterface, reflecting naming in FOS 2.x:
https://github.com/FriendsOfSymfony/FOSHttpCache/blob/master/src/ResponseTagger.php

Current one reflect naming in 1.4:
https://github.com/FriendsOfSymfony/FOSHttpCache/blob/1.4/src/Handler/TagHandler.php